header.html

```diff
@@ -1,10 +1,87 @@
+      </div>
+      <div class="site-title">i5k Workspace@NAL</div>
+   </div>
+   <ul class="sf-menu sf-js-enabled sf-arrows" id="example">
+      <li><a href='https://i5k.nal.usda.gov/'>Home</a></li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/species'>Organisms</a>
+      </li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/'>Data</a>
+         <ul>
+            <li><a href='https://i5k.nal.usda.gov/content/data-downloads'>Data Downloads</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/data-submission-overview'>Submit Data</a>
+               <ul>
+                  <li><a href='https://i5k.nal.usda.gov/datasets/submit-a-dataset'>Submit a dataset</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/data-submission-overview'>Data Submission Overview</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/datasets/request-project'>Request a new i5k Workspace Project</a></li>
+                  <li class="has-sub"><a href='https://i5k.nal.usda.gov/image_permission/new'>Sign an Image Permission Form</a></li>
+               </ul>
+            </li>
+         </ul>
+      </li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/'>Tools</a>
+         <ul>
+            <li><a href='https://i5k.nal.usda.gov/blast'>BLAST</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/'>JBrowse/Apollo</a>
+               <ul>
+                  <li><a href='https://i5k.nal.usda.gov/web-apollo-registration'>Apollo registration form</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/available-genome-browsers'>JBrowse/Apollo Organisms</a></li>
+               </ul>
+            </li>
+            <li><a href='https://i5k.nal.usda.gov/legacy_blast'>Legacy BLAST</a></li>
+            <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/webapp/hmmer/'>HMMER (beta)</a></li>
+            <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/webapp/clustal'>Clustal (beta)</a></li>
+            <li class="has-sub">
+               <a href='https://i5k.nal.usda.gov/'>Training tools</a>
+               <ul>
+                  <li><a href='https://i5k.nal.usda.gov/jbrowseapollo-training'>JBrowse/Apollo for training</a></li>
+                  <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/training/webapp/blast/'>Training BLAST</a></li>
+               </ul>
+            </li>
+         </ul>
+      </li>
+      <li class="has-sub">
+         <a href='https://i5k.nal.usda.gov/'>Tutorials and Resources</a>
+         <ul>
+            <li><a href='https://i5k.nal.usda.gov/content/blast-tutorial'>BLAST tutorial</a></li>
+            <li><a href='https://i5k.nal.usda.gov/https://i5k.nal.usda.gov/webapp/clustal/manual/'>Clustal Tutorial</a></li>
```

Comment:
This is the place that I mentioned in today's meeting. It should be directly changed in our upstream tripal site, though.